src/UI/Renderer.php

```diff
@@ -10,12 +10,14 @@
  * @author	Richard Klees <richard.klees@concepts-and-training.de>
  */
 interface Renderer {
+
 	/**
-	 * Render given component.
+	 * Render given component. If an array of components is passed, this method returns an array containing the output
+	 * of the rendered components, same index.
+	 *
+	 * @param Component\Component|Component\Component[] $component
 	 *
-	 * @param	$component			Component
-	 * @throws	\LogicException		if renderer is called with a component it can't render
-	 * @return	string
+	 * @return string|array The output of the renderer rendering the component
```

Comment:
@klees So if multiple components are passed as array, the method returns a concatenated string of each renderer, right? This would also simplify my examples 👍  :)
I guess we do not need another method then?